sympy/functions/elementary/exponential.py

```diff
@@ -19,6 +19,7 @@
 from sympy.functions.elementary.miscellaneous import sqrt
 from sympy.ntheory import multiplicity, perfect_power
 from sympy.ntheory.factor_ import factorint
+from sympy.core.symbol import Symbol
```

Comment:
Could you get rid of this unused import and let the CI run again ?
I would like to see if the tests pass or not !

Reply:
done